test/Sentry.Tests/SentrySdkTests.cs

```diff
@@ -139,7 +139,7 @@ public void Init_InvalidDsnEnvironmentVariable_Throws()
                 });
         }
 
-        [Fact]
+        [Fact(Skip = "Setting an environment variable to an empty string is the same as not having that variable set at all, so this is not testable")]
```

Comment:
As mentioned in Discord, setting an environment variable to `""` has the same effect as not setting the environment variable at all (or removing it if it was set previously).

![image](https://user-images.githubusercontent.com/1935960/123851363-efec4200-d8cf-11eb-8f31-da439c8bfdc6.png)

Works as expected with a non-empty string:

![image](https://user-images.githubusercontent.com/1935960/123851389-f8447d00-d8cf-11eb-99e8-2c21ff58c843.png)